### PR TITLE
Cut 0.1.7

### PR DESCRIPTION
**Pull request by kgang**

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -15,12 +15,8 @@ astroid = "==3.2.3"
 astor = "==0.8.1"
 celery = "==5.4.0"
 sqlalchemy = "==2.0.34"
-alembic = "==1.13.2"
 sqlmodel = "==0.0.22"
-
-
-[tool.poetry.group.tooluse.dependencies]
-boto3 = "==1.35.3"
+boto3 = "==1.35.3"  # TODO: Separate into extras
```
Comment (kgang) [inline pyproject.toml:19]: @MichaelDeng03 all dependency groups that are not main are treated as dev dependencies. We have to use their 'extras' functionality to do what we intended and allow optional install of tools modularly

**Comment by MichaelDeng03** [comment]
what happened to 0.1.6?

**Comment by kgang** [comment]
> what happened to 0.1.6?

We had to change pyproject.toml so it'd install as expected

**Comment by kgang** [comment]
We ran into an issue with `poetry run python -m concrete prompt "create a simple hello world app"`

**Pull request merged**